src/composables/authHandler.ts

```diff
@@ -3,7 +3,7 @@ import router from '../router'
 
 export const isAuthError = ref(false)
 export const isPopupShown = ref(false)
-export const handleAuthResponse = (response: boolean) => {
+export const handleAuthResponse = (response: boolean | string) => {
```

Comment:
Yeah, because when there is an error in the auth response, I want the respective function to return the error code (which happens to be a string), based on which I can display proper error messages for a user, that you recently appreciated 😛